ibis/pyspark/tests/test_array.py

```diff
@@ -0,0 +1,231 @@
+import pandas as pd
+import pandas.util.testing as tm
+import pytest
+from pytest import param
+
+import ibis
+
+pytest.importorskip('pyspark')
+pytestmark = pytest.mark.pyspark
+
+
+@pytest.fixture(scope='session')
+def client():
```

Comment:
Can we use the existing fixture for this?

Reply:
Which existing fixture?

Reply:
There is a client fixture in test_basic.py, you can move it to conftest.py and share between the two

Reply:
Done